src/compat.cpp

```diff
@@ -93,6 +93,8 @@ static void compatInit(Compatibility_t &c)
     c.luna_enable_engine = Compatibility_t::LUNA_ENGINE_UNSPECIFIED;
     c.fix_fairy_stuck_in_pipe = true;
     c.world_map_fast_move = false;
+	// 1.3.7
```

Comment:
btw, as @ds-sloth noticed, you don't need this line, simply add it as 1.3.6, because this version is still in development and wasn't released yet.